### ISSUE TITLE
Update header UI

### ISSUE THREAD

**Issue by kristinashu** [mozilla/network-pulse#233]
Update header so it is more web app like. Add logo, consider making search smaller, consider adding tagline...

Related to:
Close button issue https://github.com/mozilla/network-pulse/issues/206
Splash page (might not be necessary anymore) https://github.com/mozilla/network-pulse/issues/205

**Comment by kristinashu** [comment]
@xmatthewx this is something I had mocked up a while ago, any thoughts?

<img width="320" alt="screen shot 2016-11-17 at 10 20 48 am" src="https://cloud.githubusercontent.com/assets/1068783/20401969/b2e07418-acaf-11e6-804e-4021d4ae7d3f.png">

**Comment by xmatthewx** [comment]
I definitely prefer Favs below. And I'd prefer to keep the Add New button as we currently have it for the sake of hierarchy. Is there a problem with the current button? 

I'm less certain of this, but I think the beauty of the pink nav bar on our current prototype might be diminished with the light Mozilla there. I wonder if the header title in dark gray would be better. Maybe you tried that already?

**Comment by kristinashu** [comment]
Cool, I prefer Favs below as well. Nothing against the current add button, just trying out different things but I'll bring it back. Will keep exploring.

**Comment by kristinashu** [comment]
Still exploring, but here's a nav version very similar to the schedule app.

![empty favs copy 9 1x](https://cloud.githubusercontent.com/assets/1068783/20493763/f693dc8c-afce-11e6-9ffc-6d1ffd2ae311.png)

**Comment by kristinashu** [comment]
Or, left aligned and removed 'Featured' since clicking on the logo covers that:

<img width="321" alt="screen shot 2016-11-21 at 3 50 04 pm" src="https://cloud.githubusercontent.com/assets/1068783/20505485/39209c1a-b002-11e6-907f-2d110488dfde.png">

**Comment by kristinashu** [comment]
Got feedback from everyone and this is where we landed. Will keep iterating tho. 

<img width="320" alt="screen shot 2016-11-21 at 4 30 02 pm" src="https://cloud.githubusercontent.com/assets/1068783/20506280/caf5168e-b007-11e6-9fdb-1de68229b704.png">

**Comment by xmatthewx** [comment]
This is looking good. 

Wondering... should we shorten this to `Mozilla Pulse`? Or is it important to set an explicit network context? Let's stick with this for now, but think about it. 

Do other products use `My` in headings and nav items? I'm not opposed. Actually, I think this is a good move. It's a quiet call to action to start curating.

**Comment by kristinashu** [comment]
Mocked up on all the screens https://redpen.io/p/nzcd35d57dd04ad6dc

**Comment by mmmavis** [comment]
I think this is done: https://pulse-react.herokuapp.com/featured 😄  Feel free to open a new ticket if anything needs to be tweaked.
